app/src/Game.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Board from './Board';
+
+export const TurnContext = React.createContext();
+export const SetTurnContext = React.createContext();
```

Comment:
One other note - you can include multiple pieces of state on a single context, so I'd probably group both the turn and the setTurn function onto one TurnContext instead of separating them out.